Installation/SecurityGroup/anonymous.securityGroup.json

```diff
@@ -1,13 +1,13 @@
 {
   "$id": "https://docs.commongateway.nl/securityGroup/default.anonymous.securityGroup.json",
   "$schema": "https://docs.commongateway.nl/schemas/SecurityGroup.schema.json",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "title": "Anonymous SecurityGroup",
   "scopes": [
     "schemas.https://opencatalogi.nl/oc.catalogi.schema.json.GET",
     "schemas.https://opencatalogi.nl/oc.application.schema.json.GET",
     "schemas.https://opencatalogi.nl/oc.component.schema.json.GET",
-    "schemas.https://opencatalogi.nl/oc.organisation.schema.json.GET",
+    "schemas.https://opencatalogi.nl/oc.organization.schema.json.GET",
```

Comment:
Keep in mind that this overwrites existing securityGroup data, including scopes from other bundles. Will require some manual changes.